src/libelektra/mount.c

```diff
@@ -444,6 +444,8 @@ Backend* elektraMountGetBackend(KDB *handle, const Key *key)
 {
 	if (!key || !strcmp(keyName(key), "")) return handle->defaultBackend;
 
+	if (!handle) return NULL; // TODO: what is the best way to handle this
```

Comment:
Does this happen? Can you post the stack strace? Basically, there should be a docu that handle must be valid and an assertion that this won't happen.